apps/mobile/src/common/status-bar.tsx

```diff
@@ -0,0 +1,13 @@
+import { StatusBar as NativeStatusBar } from 'react-native';
```

Comment:
Could this live in `components`? We could pass in the `barStyle`

Reply:
Definitely. No idea why I placed this here.